dataset/api.go

```diff
@@ -171,6 +172,11 @@ func (api *API) callDatasetAPI(ctx context.Context, method, path string, payload
 	return jsonBody, resp.StatusCode, nil
 }
 
+// GetHealthCheckClient returns a healthcheck-compatible client
+func (api *API) GetHealthCheckClient() *datasetclient.Client {
+	return datasetclient.New(api.url)
```

Comment:
Consider inlining `datasetclient.New(api.url)` in main when the ticker is created.

Reply:
As discussed, I was trying to have all `datasetAPI` access in one package, and allow for a possible future merging of this client and the `go-ns` client.  Seemed cleaner to me, so hope we can agree to differ.